source/_data/plugins.yml

```diff
@@ -2816,6 +2816,13 @@
     - menu
     - categories
     - archive
+- name: Bridge
```

Comment:
![image](https://user-images.githubusercontent.com/15902347/108679723-ef116100-7527-11eb-84d6-142eda8a411e.png)


Plugin names seem to be package names, and I don't know if there's a convention for that? @hexojs/core

Reply:
Package name must start with hexo-* otherwise it won't be recognized. Given this, I thought it's ok use the real plugin name, without the hexo-* part. 
The documentation doesn't say anything about plugin names, if there's a rule that I wasn't aware of, let me know and I'll change the name.

Reply:
In a effort to unblock this PR, I renamed the plugin to match the package name.

Reply:
![image](https://user-images.githubusercontent.com/11393457/109422024-15c51100-79da-11eb-9b17-a94a41de7a55.png)